File: app/src/main/java/com/ribic/nejc/veselica/notification/PartyReminderIntentService.java
package com.ribic.nejc.veselica.notification;

import android.app.IntentService;
import android.content.Intent;
import android.support.annotation.Nullable;


public class PartyReminderIntentService extends IntentService {
    public PartyReminderIntentService() {
        super("");
    }

    @Override
    protected void onHandleIntent(Intent intent) {
        String action = intent.getAction();
        ReminderTask.executeTask(this, action);
    }
}

File: app/src/main/java/com/ribic/nejc/veselica/notification/PartyReminderIntentService.kt
package com.ribic.nejc.veselica.notification

import android.app.IntentService
import android.content.Intent


class PartyReminderIntentService : IntentService("") {

    override fun onHandleIntent(intent: Intent?) {
        val action = intent!!.action
        ReminderTask.executeTask(this, action)
    }
}
